guide/getting-started/software.md

```diff
@@ -151,7 +155,7 @@ In between these first two categories fall decentralized exchanges. They may be
    layout = "float-right-desktop"
 %}
 
-A node connects to and participates in the bitcoin network to download and exchange transaction data. Some wallets have built-in nodes and some node software also supports wallet functionality. The minimum implementation of a node however is that of syncing data and allowing other software a user has to access it.
```

Comment:
Adding comment by @sqcken for future reference, this one is about the "Mining" and "Node section.

> This feels like we need more here, it just doesn’t really describe hardware mining. The node section feels light, too.

Reply:
Could elaborate the mining bit more and detail how people who have mining hardware can choose to operate different firmware based on their needs. An example of this is braiins OS+ https://braiins.com/os/plus

Reply:
We should probably open an issue on how to expand those.

Reply:
We have a full [page about nodes](https://bitcoin.design/guide/glossary/node/) now under the Glossary.

Not sure how much we need about mining as it's not super relevant to using bitcoin as a currency. The tech primer includes [a bit about mining](https://bitcoin.design/guide/getting-started/technology-primer/#how-are-transactions-confirmed), could we just link to that?

Reply:
Added a link https://github.com/BitcoinDesign/Guide/pull/437/commits/2a8065210dd4b367035f82cd0cbc35e444b39a95

For mining, since Tech Primier is being worked on, I would avoid adding extra links to it atm, let's get back to it once we're sure how it'll shape up.